File: src/Marvin.Products.Management/Modification/Model/ProductCustomization.cs
// Copyright (c) 2020, Phoenix Contact GmbH & Co. KG
// Licensed under the Apache License, Version 2.0

using System.Collections.Generic;
using System.Runtime.Serialization;

namespace Marvin.Products.Management.Modification
{
    [DataContract]
    internal class ProductCustomization
    {
        [DataMember]
        public ProductDefinitionModel[] ProductTypes { get; set; }

        [DataMember]
        public RecipeDefinitionModel[] RecipeTypes { get; set; }

        [DataMember]
        public ProductImporter[] Importers { get; set; }
    }
}

File: src/Moryx.Products.Web/src/app/api/models/product-customization.ts
/* tslint:disable */
/* eslint-disable */
import { ProductDefinitionModel } from './product-definition-model';
import { ProductImporter } from './product-importer';
import { RecipeDefinitionModel } from './recipe-definition-model';
export interface ProductCustomization {
  importers?: null | Array<ProductImporter>;
  productTypes?: null | Array<ProductDefinitionModel>;
  recipeTypes?: null | Array<RecipeDefinitionModel>;
}
